app/models/user.rb

```diff
@@ -9,41 +9,41 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
-  has_many :received_invitations, class_name: 'FriendshipInvitation', foreign_key: 'invitee_id'
-  has_many :sent_invitations, class_name: 'FriendshipInvitation', foreign_key: 'inviter_id'
+  has_many :inverted_friendships, class_name: 'Friendship', foreign_key: 'friend_id', dependent: :destroy
+  has_many :friendships, class_name: 'Friendship', foreign_key: 'user_id', dependent: :destroy
 
   def friends
-    friends = received_invitations.map { |invitation| invitation.inviter if invitation.confirmed == true }
-    friends += sent_invitations.map { |invitation| invitation.invitee if invitation.confirmed == true }
-    friends.compact
+    friendships.map { |invitation| invitation.friend if invitation.confirmed == true }.compact
   end
 
-  def pending_invitations
-    sent_invitations.map { |invitation| invitation.invitee if invitation.confirmed == false }.compact
+  def pending_friends
+    friendships.map { |invitation| invitation.friend if invitation.confirmed == false }.compact
   end
 
   def friendship_requests
-    received_invitations.map { |invitation| invitation.inviter if invitation.confirmed == false }.compact
+    inverted_friendships.map { |invitation| invitation.user if invitation.confirmed == false }.compact
   end
 
   def confirm_friend(user)
-    friend_invitation = received_invitations.find { |invitation| invitation.inviter == user }
-    friend_invitation.confirmed = true
-    friend_invitation.save
+    friendship = inverted_friendships.find { |invitation| invitation.user == user }
+    friendship.confirmed = true
+    friendship.save
+    inverted_friendship = Friendship.new(user_id: friendship.friend_id, friend_id: friendship.user_id, confirmed: true)
+    inverted_friendship.save
   end
```

Comment:
- [ ] when friendship is confirmed, an additional row with reversed user-friend pair should be created.
```ruby
def confirm_friend
  self.update_attributes(confirmed: true)
  Friendship.create!(friend_id: self.user_id,
                  user_id: self.friend_id,
                  confirmed: true)
end
```